core/frontend/src/tile/ImdlReader.ts

```diff
@@ -142,12 +142,22 @@ interface ImdlTextureMapping {
     mode?: TextureMapping.Mode;
     /** @see [TextureMapping.Params.worldMapping]($common). Default: false. */
     worldMapping?: boolean;
+    /** @see [TextureMapping.Params.useConstantLod]($common). Default: false. */
+    useConstantLod?: boolean;
+    /** Describes the [TextureMapping.ConstantLodParamProps]($common). */
+    constantLodParams?: {
```

Comment:
Is this intentionally "Params" instead of "Props"? (It seems confusing to me that this has a different name than is used elsewhere.)

Reply:
I do see "normalMapParams" below, so it's entirely possible that "Params" is more appropriate here. If that is the case, please resolve this comment.

Reply:
Travis, you have already reviewed these changes in #5426. This PR simply merges them from the imodel02 branch into master.

Reply:
Sorry for missing this (and the extract-api problem) in my initial review. As I pointed out in my second comment, this use of "Params" does seem reasonable here, due to the other "Params" value below.